test/e2e-skipper.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+echo '>>> Loading Flagger image'
+kind load docker-image test/flagger:latest
+
+echo '>>> Installing Skipper Ingress, Flagger and Prometheus'
+# use kustomize to avoid compatibility issues:
+# https://github.com/kubernetes-sigs/kustomize/issues/2390
+# Skipper will throw an Prometheus warning which can be ignored:
+# https://github.com/weaveworks/flagger/issues/664
+
+# installing kustomize if not installed
+if ! command -v kustomize &> /dev/null; then
```

Comment:
Please remove the kustomize install and rebase with master. In the latest master, Kustomize is installed in CI.

Reply:
Thank you for that!

Reply:
adjusted